zapisy/apps/api/rest/v1/serializers.py

```diff
@@ -82,7 +83,12 @@ def to_representation(self, value):
         return value.name
 
     def to_internal_value(self, data):
-        return Program.objects.get(name=data)
+        try:
+            program = Program.objects.get(name=data)
+        except ObjectDoesNotExist:
+            raise serializers.ValidationError("Program o podanej nazwie nie istnieje.")
+        else:
+            return program
```

Comment:
Czy to nie to samo?
```suggestion
        try:
            return Program.objects.get(name=data)
        except ObjectDoesNotExist:
            raise serializers.ValidationError("Program o podanej nazwie nie istnieje.")
```
Dodatkowo, można chyba łapać `Program.DoesNotExist` (czyli niby nieco węższą klasę wyjątków), a jak zrobić to we wszystkich trzech miejscach, to nowy import przestaje być potrzebny.